### DISCUSSION TITLE
Source code for these packages (Prism.Container.Abstractions and Prism.Container.DryIoc)

### DISCUSSION THREAD

**Discussion by chk040399** [PrismLibrary/Prism#3266] [General]
Where to find the Source code for these packages (Prism.Container.Abstractions and Prism.Container.DryIoc). the source link in the nuget pages points to this URL (https://github.com/PrismLibrary/Prism.Containers) this link is not available. As the PRISM Library is open source , I thought it was the case for these packages.
I need to the source access in order to compile unsigned version of these assemblies, may be I can adress the issue https://github.com/PrismLibrary/Prism/issues/3235

**Comment by brianlagunas** [comment]
Those packages are not open to the public. The Prism library is "source available" and operates under community and commercial licenses.

https://github.com/PrismLibrary/Prism/blob/master/LICENSE
